examples/timeline/markers/customTimeMarkersEditable.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE HTML>
```

Comment:
This example file needs to be added to the [example overview](https://github.com/visjs/vis-timeline/blob/master/examples/timeline/index.html).

Reply:
Fixed in 4a52f8b